### PR TITLE
T115 validate superpowers

### PR DESCRIPTION
**Pull request by denl95**
https://trello.com/c/asWGCKEZ/115-2-t115-validate-user-s-superpowers

### CODE REVIEW

File: app/imports/startup/client/mixins.js [hunk 1]
```diff
@@ -150,6 +151,18 @@ ViewModel.mixin({
       return !!Meteor.userId() || Meteor.loggingIn();
     }
   },
+  roles: {
+    isInRole(user, orgId, role) {
+      if (user) {
+        const { [orgId]: orgRoles } = user.roles;
+
+        return orgRoles && _.contains(orgRoles, role);
```
Comment (meyginix) [inline app/imports/startup/client/mixins.js]: Why didn't you use ```Roles.userIsInRole``` here?

File: app/imports/ui/components/dashboard/dashboard-footer.js [hunk 2]
```diff
@@ -1,8 +1,21 @@
 import { Template } from 'meteor/templating';
 import { Organizations } from '/imports/api/organizations/organizations';
+import { Roles } from 'meteor/alanning:roles';
+
+import { UserRoles } from '/imports/api/constants.js';
 
 Template.DashboardFooter.viewmodel({
-  mixin: ['modal'],
+  mixin: ['modal', 'organization'],
```
Comment (meyginix) [inline app/imports/ui/components/dashboard/dashboard-footer.js]: Why didn't you use mixin ```roles``` here?

File: app/imports/startup/client/mixins.js (hunk 1 above)
Comment (denl95) [inline app/imports/startup/client/mixins.js]: @meyginix I should delete this. Because I didn't see that you are using meteor package for roles already

**Comment by Praxie** [comment]
screen shot 2016-05-13 at 12 58 19 pm
api/organizations/methods.js
you need to check invite superpower here

**Pull request closed without merging**